merlin/utils.py

```diff
@@ -57,6 +57,27 @@
 DEFAULT_FLUX_VERSION = "0.13"
 
 
+def get_priority(api="rabbit", priority="low"):
+    apis = ["rabbit", "redis"]
+    if api not in apis:
+        LOG.error(f"Unrecognized api '{api}'! Options: {apis}")
+    priorities = ["high", "mid", "low"]
+    if priority not in priorities:
+        LOG.error(f"Unrecognized priority '{priority}'! Options: {priorities}")
+    if priority == "mid":
+        return 5
+    if api == "rabbit":
+        if priority == "low":
```

Comment:
Yeah they go backwards for some insane reason.

Reply:
In case you're wondering -why- it does this, its because redis doesn't have true message priorities. Instead, it splits each queue into n distinct queues and workers go through them in alphabetical order, so 0 comes first and 9 last. We also need to modify app.conf.broker_transport_options (see https://docs.celeryproject.org/en/latest/userguide/routing.html#redis-message-priorities)